zyte_api/aio/client.py

```diff
@@ -145,6 +145,8 @@ def request_parallel_as_completed(self,
         the number of connections.
         """
         sem = asyncio.Semaphore(self.n_conn)
+        if session is None:
+            session = create_session(n_conn=self.n_conn)
```

Comment:
Does this mean the context manager interface is not used for this session instance? I wonder if it's problematic.

Reply:
I think https://github.com/zytedata/python-zyte-api/pull/59/commits/d33bc6632bd5c2f38e7dbb12f38471f1569a6a4e does the job.

That said, I will create a separate PR to implement an async session context manager in line with #22.